scripts/create_cu_vm.py

```diff
@@ -0,0 +1,121 @@
+
+    logger.info('Setting up cron jobs on the CU VM')
+
+    command_run(ssh_client, "cd /etc/init.d; wget {}/cu-disk-script.sh".format(url),
+        "CU: There was an error downloading disk script file")
+    command_run(ssh_client, "cd /etc/init.d; wget {}/cu-network-script.sh".format(url),
+        "CU: There was an error downloading network script file")
+    command_run(ssh_client,
+        "chmod +x /etc/init.d/cu-disk-script.sh /etc/init.d/cu-network-script.sh",
+        "CU: There was an error running chmod")
+    command_run(ssh_client, "cd /tmp; wget {}/crontab.in".format(url),
+        "CU: There was an error downloading crontab.in")
+    command_run(ssh_client, "crontab /tmp/crontab.in",
+        "CU: There was an error running crontab:")
+    command_run(ssh_client, "yum install -y iperf",
+        "CU: There was an error installing iperf")
+    command_run(ssh_client, "yum install -y pv",
+        "CU: There was an error installing pv")
+
+    # The cron jobs run at reboot.Hence,a reboot is required.
+    logger.info('Rebooting vm after setting up cron jobs')
+    command_run(ssh_client, "reboot &",
+        "CU: There was an error running reboot")
+
+
+def vm_running(provider, vm_name):
+    if provider.is_vm_running(vm_name):
+        logger.info("VM %s is running" % vm_name)
+    else:
+        logger.error("VM is not running")
+        return 10
+
+
+def cu_vm(provider, vm_name, template):
+    """
+    Deploys CU VM
+    """
+    # deploy_template(provider, vm_name, template, purpose)
+    provider_dict = cfme_data['management_systems'][provider]
+    datastore = provider_dict['cap_and_util']['allowed_datastores']
+    resource_pool = provider_dict['cap_and_util']['resource_pool']
+
+    deploy_template(provider, vm_name, template,
+        resourcepool=resource_pool, allowed_datastores=datastore)
+
+    prov_mgmt = get_mgmt(provider)
+    vm_running(prov_mgmt, vm_name)
+    ip = prov_mgmt.get_ip_address(vm_name)
+
+    vm_ssh_creds = provider_dict['capandu_vm_creds']
+    sshname = credentials[vm_ssh_creds]['username']
+    sshpass = credentials[vm_ssh_creds]['password']
+
+    # Create cron jobs to generate disk and network activity on the CU VM.
+    ssh_client = make_ssh_client(ip, sshname, sshpass)
+    config_cu_vm(ssh_client)
+    vm_running(prov_mgmt, vm_name)
+
+
+def main():
+    parser = argparse.ArgumentParser(description=__doc__,
+        formatter_class=argparse.RawDescriptionHelpFormatter)
+
+    parser.add_argument('--provider', help='provider key in cfme_data')
+    parser.add_argument('--template', help='the name of the template to clone')
+    parser.add_argument('--vm_name', help='the name of the VM to create', default=None)
+
+    args = parser.parse_args()
+
+    if args.vm_name is None:
+        cu_vm(args.provider, 'cu-24x7', args.template)
+        cu_vm(args.provider, 'cu-9-5', args.template)
+    else:
+        cu_vm(args.provider, args.vm_name, args.template)
```

Comment:
I wonder if you can do like:

```python
cu_vms = []

# ...
cu_vms.append(cu_vm(...))  # For every vm you generate

print "\n".join(cu_vms)  # without import json of course
```

This however presumes you modify the `cu_vm` to return the result of `deploy_template()` (deploy_template always returns the deployed vm/instance's name).

Then this can be also used in bash scripting.

```bash
for vm in `./scripts/create_cu_vm.py --params` ;
do
  # things
done 
```

What do you say @psav?